### DISCUSSION TITLE
Default to Tracks when (1 items)?

### DISCUSSION THREAD

**Discussion by brokengoose** [beetbox/beets#4254] [Q&A]
Is there a way to tell "beet import" to default to "as Tracks" when there's a directory with only 1 song in it? That is, when it prints /path/to/song (1 items).

I have a lot of singles in that format, but they're mixed in with my regular collection. I don't want to default to singleton for the entire import, but I have hundreds of full albums and hundreds of individual tracks. "as Tracks" is what I want so often that I'm willing to fix the one or two times when that's not the case.

**Comment by sampsyo** [comment]
I don't think there is an option like that at the moment, no. Sounds interesting! The use case is pretty straightforward, but FWIW, I'm somewhat concerned that the implementation might be surprisingly complex…

**Comment by JOJ0** [comment]
@brokengoose here's another thread about the "singleton or not" topic: https://github.com/beetbox/beets/issues/1167
